125_valid_palindrome/step3.cpp

```diff
@@ -0,0 +1,27 @@
+class Solution {
+private:
+    std::string to_lower_alnum(std::string s) {
```

Comment:
```suggestion
    std::string to_lower_alnum(const std::string& s) {
```

それかmodern C++ ではstring_viewを渡すことが多いかと思います。

Reply:
あ、なるほど、string_view はこういうので使えるんですね。